doc/30-namingscheme.md

```diff
@@ -0,0 +1,80 @@
+# Naming scheme
+This document will provide basic informations the naming scheme used within this project. It is essential to read, before contributing.
+
+### Basics
+The basics of the naming scheme are pretty easy and should be comprehended after taking a first glance at one of the config files.
+
+The **id** should be like this *icinga_NAMEOFLOGMESSAGE*, aswell as **add_tag**.
+The **add_field** section with **[icinga][eventtype]** always equals the name provided in the **id**, with _ to help with the readability.
+Every **grok filter** will get the following **tags_on_failure** _grokparsefailure and icinga_NAMEOFLOGMESSAGE_failed.
+```sh
+Example:
+id => "icinga_NAMEOFLOGMESSAGE"
+add_tag => "icinga_NAMEOFLOGMESSAGE"
+tag_on_failure => ["_grokparsefailure","icinga_NAMEOFLOGMESSAGE_failed"]
+add field => {
+  "[icinga][eventtype]" => "NAME_OF_LOG_MESSAGE"
+}
+```
+
+### Fieldnames
+
+The basic naming scheme is **[icinga]** + a constructed field name based on the *use* of the field and *context*. Below are examples for fieldnames by use, all of these are currently (05/23/2019) used and you are encouraged to use those. More fieldnames will make the naming scheme more and more complex.
```

Comment:
Date should be removed, because documentation should always be current state and adjusted in pull-requests when needed.

Reply:
Deleted the date.